website/content/api-docs/system/managed-keys.mdx

```diff
@@ -132,6 +132,44 @@ $ curl \
   a read-write session to the HSM. This is a boolean expressed as a string (e.g.
   `"true"`). This key is mainly to work around a limitation within AWS's CloudHSM v5
   pkcs11 implementation.
+  
+#### AWS KMS Parameters
+- `access_key` `(string: <required>)`: The AWS access key to use. This may also be provided in the
+`AWS_ACCESS_KEY_ID` environment variable or as part of the AWS profile from the AWS CLI or instance
+profile.
+
+- `curve` `(string: "P256")`: The curve to use for an ECDSA key. This field is used when
```

Comment:
There is no actual default value for `curve`: 
https://github.com/hashicorp/vault-enterprise/blob/0984b99ab637d21b4118950c90406ed4731cc161/vault/logical_system_managed_keys_ent.go#L344-L347

Reply:
This is handled in the parsing for the key type. If a curve isn't supplied for an ECDSA key, it just returns `ECDSA_P256` as the key type